render.sh

```diff
@@ -1,24 +1,132 @@
 #!/bin/bash
 
-if ! [[ -e .install/fuseki.jar ]] ; then
-	mkdir .install
-	echo "Downloading RDF store (Apache Fuseki server)..."
-	curl https://repo1.maven.org/maven2/org/apache/jena/jena-fuseki-server/3.9.0/jena-fuseki-server-3.9.0.jar -o .install/fuseki.jar
+# The parts of the TD specification encoded in RDF are rendered using STTL,
```

Comment:
Maybe we can add a short requirements statement before that to specify the needs..

### Requirements:
* Java
* graphviz  (sudo apt install graphviz )